packages/office-ui-fabric-react/src/components/TextField/MaskedTextField/MaskedTextField.tsx

```diff
@@ -90,11 +90,13 @@ export class MaskedTextField extends BaseComponent<ITextFieldProps, IMaskedTextF
   }
 
   public componentWillReceiveProps(newProps: ITextFieldProps) {
-    if (newProps.mask !== this.props.mask) {
+    if (newProps.mask !== this.props.mask || newProps.value !== this.props.value) {
```

Comment:
Can you add a test that validates that passing in a new value correctly updates the state?

Reply:
I will. 👍